### ISSUE TITLE
POST /sentiment endpoint

### ISSUE THREAD

**Issue by santiment-admin** [santiment/santiment-backend#2]
## POST /sentiment
Parameters:

```
{
  userId: string
  currency: string, // "BTC", "ETH", etc
  sentiment: string, // "bullish", "catish", "bearish"?
  date: string, // ISO 8601 format e.g. "2017-03-16T11:40:32+00:00", 
}
```

Response

```
Response code and

{
  // empty
}
// or
{
  error: string
}
```

**Comment by tzanko-matev** [comment]
We change "currency" to "asset" which is more general

**Comment by santiment-admin** [comment]
Tried to post sentiment:
```
{
    "id": "et",
    "date": "2017-04-04T00:00:00.000Z",
    "asset": "BTC/USD",
    "sentiment": "bullish"
}
```

Got error:
```
"Invalid Asset provided. Valid options are:EUR,BTC"
```

We should accept any asset on backend which user is capable of sending from mobile. Otherwise we'll need to update backend code every time an exchange adds an asset to their API.

Also, we switched from displaying priced for an asset in USD to displaying asset pairs (symbols), e.g. `BTC_USD`, `BTC_ETH`, `ETH_USD`
